### ISSUE TITLE
Setting spanCount via XML causes erratic scrolling behavior

### ISSUE THREAD

**Issue by Stonos** [rubensousa/DpadRecyclerView#162]
If you set the span count using the `app:spanCount` XML attribute instead of calling `DpadRecyclerView.setSpanCount()`, then scrolling will become buggy: If you hold down the up/down key, then scrolling will never stop.

Steps to reproduce:

1. Apply the following patch:
```diff
diff --git a/sample/src/main/java/com/rubensousa/dpadrecyclerview/sample/ui/screen/grid/GridFragment.kt b/sample/src/main/java/com/rubensousa/dpadrecyclerview/sample/ui/screen/grid/GridFragment.kt
index b0ea37b..90f6d91 100644
--- a/sample/src/main/java/com/rubensousa/dpadrecyclerview/sample/ui/screen/grid/GridFragment.kt
+++ b/sample/src/main/java/com/rubensousa/dpadrecyclerview/sample/ui/screen/grid/GridFragment.kt
@@ -78,7 +78,6 @@ class GridFragment : Fragment(R.layout.screen_recyclerview) {
 
     private fun setupRecyclerView(recyclerView: DpadRecyclerView) {
         recyclerView.apply {
-            setSpanCount(spanCount)
             setReverseLayout(args.reverseLayout)
             addOnViewHolderSelectedListener(object : OnViewHolderSelectedListener {
                 override fun onViewHolderSelected(
diff --git a/sample/src/main/java/com/rubensousa/dpadrecyclerview/sample/ui/screen/grid/GridViewModel.kt b/sample/src/main/java/com/rubensousa/dpadrecyclerview/sample/ui/screen/grid/GridViewModel.kt
index 36db61e..55e0706 100644
--- a/sample/src/main/java/com/rubensousa/dpadrecyclerview/sample/ui/screen/grid/GridViewModel.kt
+++ b/sample/src/main/java/com/rubensousa/dpadrecyclerview/sample/ui/screen/grid/GridViewModel.kt
@@ -13,7 +13,7 @@ class GridViewModel : ViewModel() {
     private val list = ArrayList<Int>()
     private val listLiveData = MutableLiveData<MutableList<Int>>()
     private val loadingStateLiveData = MutableLiveData<Boolean>()
-    private val pageSize = 50
+    private val pageSize = 5000
 
     val loadingState: LiveData<Boolean> = loadingStateLiveData
     val listState: LiveData<MutableList<Int>> = listLiveData
diff --git a/sample/src/main/res/layout/screen_recyclerview.xml b/sample/src/main/res/layout/screen_recyclerview.xml
index 52cbada..d24501e 100644
--- a/sample/src/main/res/layout/screen_recyclerview.xml
+++ b/sample/src/main/res/layout/screen_recyclerview.xml
@@ -24,7 +24,8 @@
         android:id="@+id/recyclerView"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:clipChildren="false" />
+        android:clipChildren="false"
+        app:spanCount="5" />
 
     <com.rubensousa.dpadrecyclerview.sample.ui.widgets.item.SelectionOverlayView
         android:id="@+id/selectionOverlayView"
```
2. Open the sample app and go to the grid example
3. Hold down the down key
4. The RecyclerView will keep scrolling even after the down key is no longer held down

**Comment by rubensousa** [comment]
Hello @Stonos 

Thanks for the bug report. This fix will be available in `1.1.0-beta02` this weekend: https://github.com/rubensousa/DpadRecyclerView/pull/163

**Comment by rubensousa** [comment]
@Stonos released in `1.1.0-beta02`

**Comment by Stonos** [comment]
Thank you!
